### PR TITLE
Updated german translation

### PR DESCRIPTION
**Pull request by moebelwagen**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [ ] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

The german translation for thumb ratings is twisted, thumb up refers to thumb down and vice versa. This commit fixes it.

### CODE REVIEW

**Comment by CastagnaIT** [comment]
please check also the latest changes,
you can look at this commit as reference:
https://github.com/CastagnaIT/plugin.video.netflix/commit/68299320af81375b3368f9b48d56df91fa838048

**Comment by tweimer** [comment]
@CastagnaIT Is it final for version 1.0?

@Moebelwagen You want to do the latest changes or should I? I don't mind if you do it, we should just coordinate.

**Comment by CastagnaIT** [comment]
i do not have a date for the release

to first i have to wait the inputstreamhelper 0.4.4 release, i dont know when they release,
because the current inputstreamhelper version has a critical bug that prevents this addon from working correctly

in addition i'm still running some tests for the watched status sync

**Comment by tweimer** [comment]
I was just asking if there are more translation changes to be accepted.

**Comment by CastagnaIT** [comment]
No, i don't think there will be other changes

**Comment by moebelwagen** [comment]
@tweimer I can have a look at it at the weekend.

**Comment by tweimer** [comment]
Cool. There is a new version of resource.language.de_de/strings.po in the repo, so you better rebase your changes first. Then you can do it all in one PR. Take a look at the history of that file to see latest changes, the German translation must be changed accordingly. I'll review it when you're done.

File: resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -654,11 +654,11 @@ msgstr ""
 
 msgctxt "#30157"
 msgid "The configuration has been completed"
-msgstr ""
+msgstr "Die Einrichtung wurde abgeschlossen"
 
 msgctxt "#30158"
 msgid "Run add-on configuration wizard"
-msgstr ""
+msgstr "Konfigurationsassistenten für das Addon ausführen"
```
Comment (moebelwagen) [inline resources/language/resource.language.de_de/strings.po]: @tweimer #30157 and #30158 are translated.

Comment (tweimer) [reply resources/language/resource.language.de_de/strings.po]: Cool. It can be merged now.

**Comment by CastagnaIT** [comment]
good thanks

**Comment by moebelwagen** [comment]
Glad to read that, thanks! Was my first PR (-:

Am 01.03.20 um 10:56 schrieb Stefano Gottardo:

**Pull request merged**